test/JDBC/upgrade/16_6/schedule

```diff
@@ -590,3 +590,4 @@ test_conv_money_to_varchar
 fixeddecimal_modulo
 test_conv_float_to_varchar_char_before_17_4
 BABEL_OBJECT_RESOLUTION_IN_ROUTINES
+test_constraint_like-before-17-5-or-16-9
```

Comment:
should we also test some 15.x to 16.latest path?

Reply:
For major version upgrade we are already testing from 16 to 17. We do not plan to cherry-pick the change into 15X.